tests/random_test.py

```diff
@@ -206,7 +206,11 @@ def test_choice_flags(self):
                     for p in [None, weights]:
                         previous = choice_arrays.pop(0)
                         current = rng.choice(a, size, replace, p)
-                        self.assertTrue(np.allclose(previous.to_list(), current.to_list()))
+                        res = np.allclose(previous.to_list(), current.to_list())
+                        if not res:
+                            print(f"\nnum locales: {cfg['numLocales']}")
```

Comment:
I actually left this in on purpose! I was thinking that if we got a failure in the nightly this would make it easier to get a seed to reproduce. But I don't think that will be likely since it's using more or less the same functionality as `choice`

Reply:
I think it's OK if you want to leave it in.  Thanks for explaining.